lib/ssh/doc/src/ssh_client_channel.xml

```diff
@@ -175,7 +175,7 @@
       <name since="OTP 21.0">reply(Client, Reply) -> _</name>
       <fsummary>Sends a reply to a client.</fsummary>
       <type>
-        <v>Client = opaque()</v>
+        <v>Client = <seetype marker="stdlib:gen_server#from">gen_server:from()</seetype></v>
```

Comment:
Do we really want to expose that  the channel abstraction is a gen_server ?  I think this should be opaque to the ssh user.